app/views/page/_two_to_one_image_component.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="grid-row grid-gap display-flex flex-column flex-row-tablet">
+  <% if component.text_alignment == "Left" %>
+    <div class="<%= component.flipped_ratio? ? 'tablet:grid-col-8' : 'tablet:grid-col-4' %>">
+      <% if component.title.present? %>
+        <h2 class="margin-top-0">
+          <%= component.title %>
+        </h2>
```

Comment:
let's downgrade this to an `h3` so it can get nested under an h2 in this part of the design: https://www.figma.com/file/ccwLbt94U9QGfnuFCwOXzY/Core-Designs---VA-DM?type=design&node-id=6181%3A38719&mode=design&t=fQ21h3SmfXfS3P7o-1